agora/main_ui/operations/backend_api.yaml

```diff
@@ -0,0 +1,143 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: initdb-configmap
+  namespace: contoso-hypermarket
+data:
+  initdb.sql: |
```

Comment:
Why not use the file initdb.sql ?

Reply:
Good question, i couldnt get it working quickly and wanted to get this out the door today. I will revisit.

Reply:
this is addressed now and inline SQL removed